android/src/main/java/co/il/nester/android/react/streetview/NSTStreetViewManager.java

```diff
@@ -51,8 +51,8 @@ public void setPov(NSTStreetView view, ReadableMap pov) {
     public @Nullable
     Map getExportedCustomDirectEventTypeConstants() {
         return MapBuilder.of(
-                NSTStreetViewEvent.eventNameForType(NSTStreetViewEvent.ON_ERROR), MapBuilder.of("registrationName", "onStreetViewError"),
```

Comment:
It seemed to me the reason the onSuccess Callback was working is because the event names were mismatched. So with this change I made it all the same as whats in the NSTStreetViewEvent.